v1.1/admin-ui-overview.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Overview
+toc: false
+---
+
+CockroachDB Admin UI provides details about your cluster and database configuration, and helps you optimize cluster performance by monitoring: 
+
+Area | Description
+--------|----
+[Cluster Health](admin-ui-access-and-navigate.html#summary-panel) | Includes metrics such as number of Live, Dead, and Suspect nodes, Unavailable Ranges, Queries per Second, and Service Latency. 
```

Comment:
View essential metrics about the cluster's health, such as the number of live, dead, and suspect nodes, the number of unavailable ranges, and the queries per second and service latency across the cluster.